<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedReaderConfig</class>
 <widget class="QWidget" name="FeedReaderConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="updateGroupBox">
     <property name="title">
      <string>Update</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="updateIntervalLabel">
        <property name="text">
         <string>Interval in minutes (0 = manual)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="updateIntervalSpinBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="storageTimeGroupBox">
     <property name="title">
      <string>Storage time</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="storageTimeLabel">
        <property name="text">
         <string>Days (0 = off)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="storageTimeSpinBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxyGroupBox">
     <property name="title">
      <string>Proxy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="useProxyCheckBox">
        <property name="text">
         <string>Use proxy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ProxyWidget" name="proxyWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QCheckBox" name="openAllInNewTabCheckBox">
        <property name="text">
         <string>Open all feeds in new tab</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="setMsgToReadOnActivate">
        <property name="text">
         <string>Set message to read on activate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="saveInBackgroundCheckBox">
        <property name="text">
         <string>Save configuration in background (for slow systems, more memory needed)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>301</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProxyWidget</class>
   <extends>QWidget</extends>
   <header>gui/ProxyWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>updateIntervalSpinBox</tabstop>
  <tabstop>storageTimeSpinBox</tabstop>
  <tabstop>useProxyCheckBox</tabstop>
  <tabstop>proxyWidget</tabstop>
  <tabstop>saveInBackgroundCheckBox</tabstop>
  <tabstop>setMsgToReadOnActivate</tabstop>
  <tabstop>openAllInNewTabCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
